### ISSUE TITLE
Password info 

### ISSUE THREAD

**Issue by SueBee52** [fuaaaaad/Login-Panel#2]
For my hotmail & Microsoft account is missing & I can't find reset

**Comment by fuaaaaad** [comment]
Did you give any email or mobile number for backup?

**Comment by SueBee52** [comment]
Yes both when I try to recover from both the email & the mobile phone, I still can't varify. 
I have both the Authy & Microsoft authenticaters.

**Comment by SueBee52** [comment]
On the Microsoft it won't display the confirmation numbers 
On the Authy it says on Microsoft login I have the wrong code

**Comment by SueBee52** [comment]
Help please
